explorer/src/components/common/NFTArt.tsx

```diff
@@ -105,27 +105,11 @@ const VideoArtContent = ({
   files,
   uri,
   animationURL,
-  active,
 }: {
   files?: (MetadataJsonFile | string)[];
   uri?: string;
   animationURL?: string;
-  active?: boolean;
 }) => {
-  const [playerApi, setPlayerApi] = useState<StreamPlayerApi>();
-  const playerRef = useCallback(
-    (ref) => {
-      setPlayerApi(ref);
-    },
-    [setPlayerApi]
-  );
-
-  useEffect(() => {
-    if (playerApi) {
-      playerApi.currentTime = 0;
-    }
-  }, [active, playerApi]);
```

Comment:
The `active` prop isn't used anywhere in the Explorer, so all of this code is for naught. Ripping it out rather than to fix the type errors. cc/ @roederw